### ISSUE TITLE
SDL version: Turbo speed issues

### ISSUE THREAD

**Issue by Yave-Yu** [TASEmulators/fceux#683]
1. If adjusted channel volume, such as set Triangle’s volume to 0, press turbo hotkey will make it ignore setting and output full volume.
2. Each time turn on/off turbo, cause a pop click sound, it even affects output AVI/WAV.

**Comment by thor2016** [comment]
I did a quick test of these, the behavior does not match your description.
1. I adjusted various channel volumes and then pressed the turbo key. With turbo key pressed I get no audio at all. When I release the turbo key, audio comes back with same settings as before.
2. I recorded to an AVI file, pressed turbo at various times. Behavior was a described in block 1. I then played the AVI using VLC. The AVI played smoothly with no signs of turbo mode use (which is what I would expect).
In short, I cannot repro any of your issues. Please provide more detail.

**Comment by Yave-Yu** [comment]
Did tested it on win32/64 version?
